devices/awox.js

```diff
@@ -1,4 +1,73 @@
 const extend = require('../lib/extend');
+const fz = require('../converters/fromZigbee');
+const exposes = require('../lib/exposes');
+const e = exposes.presets;
+
+const awoxRemoteHelper = {
+    convertToColorName: (buffer) => {
+        const commonForColors = buffer[0] === 17 && buffer[2] === 48 && buffer[3] === 0 && buffer[5] === 8 && buffer[6] === 0;
+
+        if (commonForColors && buffer[4] === 255) {
+            return 'red';
+        } else if (commonForColors && buffer[4] === 42) {
+            return 'yellow';
+        } else if (commonForColors && buffer[4] === 85) {
+            return 'green';
+        } else if (commonForColors && buffer[4] === 170) {
+            return 'blue';
+        }
+        return null;
+    },
+    isRefresh: (buffer) => {
+        return buffer[0] === 17 && buffer[2] === 16 && buffer[3] === 1 && buffer[4] === 1;
+    },
+};
+
+const fzLocal = {
+    colors: {
+        cluster: 'lightingColorCtrl',
+        type: ['raw'],
+        convert: (model, msg, publish, options, meta) => {
+            const color = awoxRemoteHelper.convertToColorName(msg.data);
+            if (color != null) {
+                return {
+                    action: 'button_'+color,
+                };
+            }
+            return {
+                action: null,
+            };
+        },
+    },
+    refreshColored: {
+        cluster: 'lightingColorCtrl',
+        type: ['commandMoveHue'],
+        convert: (model, msg, publish, options, meta) => {
+            if (msg.data.movemode === 1 && msg.data.rate === 12) {
+                return {
+                    action: 'button_refresh_colored',
+                };
+            }
+            return {
+                action: null,
+            };
+        },
+    },
+    refresh: {
+        cluster: 'genLevelCtrl',
+        type: ['raw'],
+        convert: (model, msg, publish, options, meta) => {
+            if (awoxRemoteHelper.isRefresh(msg.data)) {
+                return {
+                    action: 'button_refresh',
+                };
+            }
+            return {
```

Comment:
In this case don't return anything (also for the other converters)

Reply:
Done :)